multichain-testing/scripts/ymax-tool.ts

```diff
@@ -118,8 +142,41 @@ const main = async (
     connectWithSigner = SigningStargateClient.connectWithSigner,
   } = {},
 ) => {
-  const [volume] = argv.slice(2);
-  if (!volume) throw Error(`USAGE: ${argv[1]} 123.45`);
+  // Parse command line arguments using node:util's parseArgs
+  const { values, positionals } = parseArgs({
+    args: argv.slice(2),
+    options: {
+      'skip-poll': {
+        type: 'boolean',
+        default: false,
+      },
+      'exit-success': {
+        type: 'boolean',
+        default: false,
+      },
+      help: {
+        type: 'boolean',
+        short: 'h',
+        default: false,
+      },
+    },
+    allowPositionals: true,
+  });
+
+  // Extract options
+  const skipPoll = values['skip-poll'];
+  const [volume] = positionals;
+
+  // Show help if requested or if volume is not provided
+  if (values.help || !volume) {
+    console.log(`USAGE: ${argv[1]} <volume> [options]
+Options:
+  --skip-poll       Skip polling for offer result
+  --exit-success    Exit with success code even if errors occur
+  -h, --help        Show this help message`);
```

Comment:
suggestion: hoist the USAGE string to the top of the file (just after imports) as documentation.
I suppose it's a function of `argv[1]` rather than a string constant. still works as doc, I think.